Rephrase "Creating Custom Fragments" Section (#2122)
MarkBind/markbind: docs/userGuide/reusingContents.md

</span>
</include>

#### Excluding Files from Rendering as Pages

> **MarkBind supports the exlusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation.

Reviewer: I think "exclusion" is misspelled as "exlusion" in the bolded sentence!

Author: Good catch! Thank you!

> **MarkBind supports the exlusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation.

Reviewer: I think since the "Reusing Contents Across Sites" section is now the parent section for this section, perhaps simply referring to it could be an alternative to linking it?

Author: I agree! I think it will look cleaner as well!

</span>
</include>

#### Excluding Files from Rendering as Pages

> **MarkBind supports the exclusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to reusing content, but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation. This facilitates the use of repeated content across the site.

Reviewer 2: @itsyme Could we perhaps rephrase this further to make it clearer? It seems a little lengthy and the link to "reusing content" has been removed, which makes this part confusing:
> This allows you to create reusable content similar to reusing content, but rather than reusing content from a _sub-site_, content is reused from fragments which are excluded from page generation.

wdyt?

Author: I agree about the lengthiness! I will rephrase it to keep it more succinct. As for the removal of the link, I initially agreed with what @lhw-1 said above, the reasoning being that this section is already under the reusing content section. However, I would be open to inserting the link back as well to make it seem clearer.

> **MarkBind supports the exclusion of files from rendering as pages**. This allows for the ease of creation of <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip>. This allows you to create reusable content similar to [reusing content](#reusing-contents-across-sites), but instead of a _sub-site_, content is reused from fragments which are excluded from page generation. This facilitates using repeated content across pages.

Reviewer 2: ```suggestion
**MarkBind supports the exclusion of files from rendering as pages**. For example, you can exclude files containing <tooltip content="A fragment is a piece of content that can be reused across multiple pages.">custom fragments</tooltip> of reusable content from page generation.
```

Reviewer 2: I think there's a typo in the original issue, which says that this part should:
> Move it ~~under~~ above the "Reusing Contents Across Sites" section, under "Includes", as it is also useful with just includes, even without subsites.

Author: Oh I see, I think this makes more sense as well! I have made the relevant changes in the latest commit!
